webhook_to_fedora_messaging/config/config.py

```diff
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from os import environ
+
+import toml
+
+from webhook_to_fedora_messaging.exceptions import ConfigError
+
+
+def get_config():
+    path = environ["W2FM_APPCONFIG"]
+    try:
+        data = toml.load(path)["flaskapp"]
+        return data
```

Comment:
Those two lines can probably be collapsed into one, but that's really a detail :-)